packages/core/package.json

```diff
@@ -114,7 +114,7 @@
     "deriv-trader": "^3.8.0",
     "deriv-translations": "^1.0.0",
     "event-source-polyfill": "^1.0.5",
-    "formik": "^1.5.8",
+    "formik": "^2.0.6",
```

Comment:
will we need to do regression testing on our forms after updating this?

Reply:
there was a mismatch linking from deriv-core to deriv-p2p, since i make formik as ext dependencies. i will make both 1.5.8 to sync them better